### DISCUSSION TITLE
Request: Resizable Widgets

### DISCUSSION THREAD

**Discussion by HoCoOhNo** [j7126/OctoPrint-Dashboard#357] [Ideas]
Thank you so much for this plugin. I run octoprint on a windows surface tablet running Ubuntu, and have it set to auto open your plugin in full mode on boot. This is a fantastic way to show all the important info right next to the printer.

My request, would it be possible to be able to resize and reposition the widgets in full mode? Some of the gauges are a little too small on my display, and scaling the website only seems to change the printer message widget. And when in full mode, with all the widgets enabled, some fall off the screen. Currently I am running it without the webcam to get all the other info to be displayed properly.

**Comment by j7126** [comment]
This will definitely be included in version 2.
